app/templates/events/view/speakers/list.hbs

```diff
@@ -1,61 +1,7 @@
 <div class="sixteen wide column">
-  <table class="ui stackable very basic table">
-    <thead>
-      <tr>
-        <th>{{t 'Name'}}</th>
-        <th>{{t 'Email'}}</th>
-        <th>{{t 'Phone'}}</th>
-        <th>{{t 'Submitted Sessions'}}</th>
-        <th>{{t 'State'}}</th>
-        <th></th>
-      </tr>
-    </thead>
-    <tbody>
-      {{#each model as |speaker|}}
-        <tr>
-          <td>
-            <a class="ui avatar round image">
-              {{widgets/safe-image src=speaker.image}}
-            </a>
-            {{speaker.name}}
-          </td>
-          <td>
-            {{speaker.email}}
-          </td>
-          <td>
-            {{speaker.phone}}
-          </td>
-          <td>
-            <div class="ui ordered list">
-              {{#each speaker.submitted_session as |session|}}
-                <div class="item">{{session}}</div>
-              {{/each}}
-            </div>
-          </td>
-          <td>
-            <div>
-              {{#if (eq speaker.state 'confirmed')}}
-                <div class="ui green label">{{t 'Confirmed'}}</div>
-              {{else}}
-                <div class="ui red label">{{t 'Canceled'}}</div>
-              {{/if}}
-            </div>
-          </td>
-          <td>
-            <div class="ui vertical compact basic buttons">
-              {{#ui-popup content=(t 'View') class='ui icon button' position='left center'}}
-                <i class="unhide icon"></i>
-              {{/ui-popup}}
-              {{#ui-popup content=(t 'Edit') class='ui icon button' position='left center'}}
-                <i class="edit icon"></i>
-              {{/ui-popup}}
-              {{#ui-popup content=(t 'Delete') class='ui icon button' position='left center'}}
-                <i class="trash outline icon"></i>
-              {{/ui-popup}}
-            </div>
-          </td>
-        </tr>
-      {{/each}}
-    </tbody>
-  </table>
+  {{events/events-table columns=columns data=model
```

Comment:
@sumedh123 the error you're getting is because, you're using events table here. Not all columns used in the events table would work with the data in the speakers response.